src/hdf5_hl.ts

```diff
@@ -76,11 +82,15 @@ function getAccessor(type: 0 | 1, size: Metadata["size"], signed: Metadata["sign
 }
 
 export type OutputData = TypedArray | string | number | bigint | boolean | (string | number | bigint | boolean | OutputData)[];
+export type JSONCompatibleOutputData = string | number | boolean | (string | number | boolean | JSONCompatibleOutputData)[];
```

Comment:
I think this should be equivalent
```suggestion
export type JSONCompatibleOutputData = string | number | boolean | JSONCompatibleOutputData[];
```

Reply:
I guess a similar simplification could be applied to OutputData...